### ISSUE TITLE
[TECHDEBT] [P2P] Documentation improvement

### ISSUE THREAD

**Issue by deblasis** [pokt-network/pocket#274]
## Objective

Tend to these TODOes:

```go
./p2p/README.md:DISCUSS(team): If you feel this needs a diagram, please reach out to the team for additional details.
./p2p/README.md:TODO(olshansky, BenVan): Link to RainTree visualizations once it is complete.
```

Have a discussion with @Olshansk to define scope and update below

## Origin Document

[ Why? Issue justification and/or link to another document]

## Goals

- _\_*REPLACE_ME*: List of things that are high level ideas or goals driving the task_
- [ ] ...

## Deliverable

- _\_*REPLACE_ME*: List of things that are concrete deliverables_
- [ ] ...

## Non-goals / Non-deliverables

- _\_*REPLACE_ME*: List of things that are out of scope_
- ...

## General issue deliverables

- [ ] Update the appropriate CHANGELOG
- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references
- [ ] If applicable, update the source code tree explanation
- [ ] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)

## [Optional] Testing Methodology

- \__REPLACE_ME_: Make sure to update the testing methodology appropriately\_
- **Task specific tests**: `make ...`
- **All tests**: `make test_all`
- **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)

---

**Creator**: @deblasis
**Co-Owners**: @Olshansk

**Comment by Olshansk** [comment]
I can help out with this one.

**Comment by Olshansk** [comment]
Let's update the README by pointing to the original presentation [here](https://drive.google.com/file/d/1PaAWTx47nPSrM798lxfKv0GnhLthZLYm/view?usp=sharing), and at some point in the future, we can update the documentation if/when we have time.
